services/galley/src/Galley/Cassandra/TeamFeatures.hs

```diff
@@ -170,24 +170,24 @@ getFeatureConfig FeatureSingletonMlsE2EIdConfig tid = do
   let q = query1 select (params LocalQuorum (Identity tid))
   retry x1 q <&> \case
     Nothing -> Nothing
-    Just (Nothing, _, _) -> Nothing
-    Just (Just fs, mGracePeriod, mUrl) ->
+    Just (Nothing, _, _, _, _) -> Nothing
+    Just (Just fs, mGracePeriod, mUrl, mCrlProxy, mUseProxyOnMobile) ->
       Just $
         WithStatusNoLock
           fs
           ( -- FUTUREWORK: this block is duplicated in
```

Comment:
Is this still applicable?

Reply:
Yes, it's still the case. I paid attention to both places.